code/obj/submachine/slots.dm

```diff
@@ -133,229 +142,51 @@
 	else
 		. = ..()
 
-/obj/submachine/slot_machine/proc/money_roll()
-	var/roll = rand(1,1400)
+/obj/submachine/slot_machine/proc/money_roll(wager)
+	var/roll = rand(1, max_roll)
 	var/exclamation = ""
 	var/win_sound = "sound/machines/ping.ogg"
-	var/amount_text = "error credits"
 	var/amount = 0
 
-	if (roll == 1)
-		command_alert("Congratulations to [src.scan.registered] on winning the Jackpot of ONE MILLION CREDITS!", "Jackpot Winner")
+	if(wager <= 250)
+		roll = max(5, roll)
+	if(src.emagged)
+		roll = max(roll, 200)
+
+	if (roll == 1) //1
 		win_sound = "sound/misc/airraid_loop_short.ogg"
 		exclamation = "JACKPOT! "
-		amount = 1000000
-		amount_text = "MILLION CREDITS"
-	else if (roll > 1 && roll <= 5)
-		command_alert("Congratulations to [src.scan.registered] on winning a hundred thousand credits!", "Big Winner")
-		win_sound =  "sound/misc/klaxon.ogg"
-		exclamation = "Big Winner! "
-		amount = 100000
-		amount_text = "a hundred thousand credits"
-	else if (roll > 5 && roll <= 25)
+		amount = 300 * wager
+		command_alert("Congratulations to [src.scan.registered] on winning a Jackpot of [amount] credits!", "Jackpot Winner")
+	else if (roll <= 5) //4
+		command_alert("Congratulations to [src.scan.registered] on winning [amount] credits!", "Big Winner")
 		win_sound =  "sound/misc/klaxon.ogg"
 		exclamation = "Big Winner! "
-		amount = 10000
-		amount_text = "ten thousand credits"
-	else if (roll > 25 && roll <= 50)
+		amount = 100 * wager
+	else if (roll <= 15) //10 or 15
```

Comment:
What's the comment here?

Reply:
10 chances if wager > 250, 15 otherwise (jackpot rolls fall through)

Reply:
Alternative phrasing: if I don't understand the comment without having to read the code, it's not a good comment and should be updated.